spec/models/category_list_spec.rb

```diff
@@ -524,5 +524,11 @@ def ordered_category_list(some_user)
       expect(cat.name).to eq("Other Name")
       expect(cat.description).to eq("Other Desc")
     end
+
+    it "safely returns categories when SiteSetting.fixed_category_positions is set" do
```

Comment:
```suggestion
    it "returns categories when SiteSetting.fixed_category_positions is enabled" do
```

Abit of "description" golfing here. I feel like "safely" is redundant and that `enabled` is much clearer than `set` when describing the state of a site setting.